### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Plou**
- Clean script integration
- Remove unused folder
- Add timberdate filter

### CODE REVIEW

File: lib/register_scripts/Classes/DummyRegisterStyles.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ class DummyRegisterStyles
     /**
      * Constructs this post
      */
-    public function __construct( $id, $path = NULL, $deps = array(), $version = '0.0.0', $inFooter = true )
+    public function __construct( $id, $path = NULL, $deps = array(), $version = false, $inFooter = false )
```
Comment (Gregcop1) [inline lib/register_scripts/Classes/DummyRegisterStyles.php]: `NULL` is the default value, so you don't have to set it, isn't it ?

File: lib/timber/add_timberdate_to_twig.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<?php
+add_filter( 'get_twig', 'add_timberdate_to_twig' );
+
+
```
Comment (Gregcop1) [inline lib/timber/add_timberdate_to_twig.php]: useless line

File: lib/timber/add_timberdate_to_twig.php [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+function add_timberdate_to_twig( $twig ) {
+    $twig->addFilter('timberdate', new Twig_Filter_Function(function($timestamp, $format) {
+       $dt = new DateTime();
+       $year = (int)date('Y', $timestamp);
+
+       if ($year < 1900 || $year > 3000) {
+           $timestamp = $timestamp / 1000;
+       }
+
+    $dt->setTimestamp($timestamp);
```
Comment (Gregcop1) [inline lib/timber/add_timberdate_to_twig.php]: indentation problem

File: lib/timber/add_timberdate_to_twig.php [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    ));
+    return $dt->format($format);
+   }));
+
+   return $twig;
+ }
+
+function wp_get_timezone_string() {
+    // if site timezone string exists, return it
+    if ( $timezone = get_option( 'timezone_string' ) )
```
Comment (Gregcop1) [inline lib/timber/add_timberdate_to_twig.php]: useless blanks between parenthesis. Standard require brackets even if there's just one line

File: lib/timber/add_timberdate_to_twig.php [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        return $timezone;
+
+    // get UTC offset, if it isn't set then return UTC
+    if ( 0 === ( $utc_offset = get_option( 'gmt_offset', 0 ) ) )
```
Comment (Gregcop1) [inline lib/timber/add_timberdate_to_twig.php]: same here

File: lib/timber/add_timberdate_to_twig.php [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        return 'UTC';
+
+    // adjust UTC offset from hours to seconds
+    $utc_offset *= 3600;
+
+    // attempt to guess the timezone string from the UTC offset
+    if ( $timezone = timezone_name_from_abbr( '', $utc_offset, 0 ) ) {
```
Comment (Gregcop1) [inline lib/timber/add_timberdate_to_twig.php]: and here. You can set your IDE to check that on save

**Comment by FeetloveR** [comment]
+1

**Comment by toofff** [comment]
+1

**Comment by Plou** [comment]
I've solve the issues you spotted. It should be ready to merge

File: web [hunk 7]
```diff
@@ -1 +1 @@
-Subproject commit ea1285f49baf9dac294ba6516e93237da54e656b
+Subproject commit a9929c4a11b95c9a239c6dd703dc0b41a8bf7908
```
Comment (Gregcop1) [inline web]: What's that ?

**Comment by Plou** [comment]
I  hate submodules

File: lib/register_scripts/Classes/DummyRegisterStyles.php (hunk 1 above)
Comment (Plou) [inline lib/register_scripts/Classes/DummyRegisterStyles.php]: It throw a warning if `DummyRegisterStyles` is called without the `$path`argument

**Pull request merged**